video/hwdec.h

```diff
@@ -55,11 +55,11 @@ void hwdec_devices_remove(struct mp_hwdec_devices *devs, struct mp_hwdec_ctx *ct
 // If used at all, this must be set/unset during initialization/uninitialization,
 // as concurrent use with hwdec_devices_request() is a race condition.
 void hwdec_devices_set_loader(struct mp_hwdec_devices *devs,
-    void (*load_api)(void *ctx), void *load_api_ctx);
+    void (*load_api)(void *ctx, int imgfmt), void *load_api_ctx);
 
 // Cause VO to lazily load all devices, and will block until this is done (even
```

Comment:
"to lazily load all devices for a specific hardware IMGFMT" or so?

Reply:
Also mentioning IMGFMT_NONE as "all" instead of "no limitations" might be a good idea.